modules/core/src/main/java/org/apache/ignite/spi/systemview/view/CacheView.java

```diff
@@ -331,7 +339,46 @@ public String cacheWriterFactory() {
 
     /** @see CacheConfiguration#getExpiryPolicyFactory() */
     public String expiryPolicyFactory() {
-        return toStringSafe(cache.cacheConfiguration().getExpiryPolicyFactory());
+        if (cache.cacheConfiguration().getExpiryPolicyFactory() == null)
+            return null;
+
+        ExpiryPolicy expiryPlc = (ExpiryPolicy)cache.cacheConfiguration().getExpiryPolicyFactory().create();
+
+        Function<Duration, String> func = (duration) -> {
+            if (duration == null)
+                return null;
+
+            return new StringBuilder()
+                .append(duration.getDurationAmount())
+                .append(' ')
+                .append(duration.getTimeUnit() == null ? TimeUnit.MILLISECONDS : duration.getTimeUnit())
+                .toString();
```

Comment:
Here is string still created, so StringBuilder is not required.
Let's handle ETERNAL somehow (compare with Duration.ETERNAL) and omit `null` components. For example: `CreatedExpiryPolicy [create=100 MILLISECONDS]`,  `EternalExpiryPolicy [create=ETERNAL]`,  etc

Reply:
Tried to reduce the number of Strings in favor of StringBuilder. Added checks for `ETERNAL` and `ZERO` durations. Refactored the logic for `ExpiryPolicy` string representation to take into account nulls